[docs] adds  steps required for email / phone links (#21316)
expo/expo: docs/pages/guides/linking.mdx

| `tel`            | Open phone app, eg: `tel:+123456789`          |
| `sms`            | Open SMS app, eg: `sms:+123456789`            |

> On newer Android versions, you may have to include the appropriate queries in your `AndroidManifest.xml` to open links (ideally via a config plugin). For example, the following:

Reviewer: ```suggestion
On newer Android versions, include the appropriate queries in the **AndroidManifest.xml** to open links. This can be done by [creating a config plugin](/config-plugins/plugins-and-mods/#create-a-plugin). For example:
```

> ```xml
> <manifest>
>   <queries>
>     <intent>
>       <action android:name="android.intent.action.SENDTO"/>
>       <data android:scheme="mailto"/>
>     </intent>
>     <intent>
>       <action android:name="android.intent.action.DIAL"/>
>     </intent>
>   </queries>
> </manifest>
> ```

Reviewer: ````suggestion
```js my-plugin.js
const { withAndroidManifest } = require("@expo/config-plugins");

const withAndroidQueries = (config) => {
  return withAndroidManifest(config, (config) => {
    config.modResults.manifest.queries = [
      {
        intent: [
          {
            action: [{ $: { "android:name": "android.intent.action.SENDTO" } }],
            data: [{ $: { "android:scheme": "mailto" } }],
          },
          {
            action: [{ $: { "android:name": "android.intent.action.DIAL" } }],
          },
        ],
      },
    ];

    return config;
  });
};

module.exports = withAndroidQueries;
```

You can then [import the custom config plugin](/config-plugins/plugins-and-mods/#import-a-plugin) in your project's app config. This example plugin will enable linking to phone and email apps.
````

Instead of referring the config and mentioning that the developer needs to create a config plugin, let's provide an example for a config plugin.

| `tel`            | Open phone app, eg: `tel:+123456789`          |
| `sms`            | Open SMS app, eg: `sms:+123456789`            |

> On newer Android versions, include the appropriate queries in the **AndroidManifest.xml** to open links. This can be done by [creating a config plugin](/config-plugins/plugins-and-mods/#create-a-plugin). For example:

Reviewer: ```suggestion
On newer Android versions, include the appropriate queries in the **AndroidManifest.xml** to open links. This can be done by [creating a config plugin](/config-plugins/plugins-and-mods/#create-a-plugin). For example, the config plugin below will enable linking to phone and email apps:
```

```js my-plugin.js
const { withAndroidManifest } = require("@expo/config-plugins");

const withAndroidQueries = (config) => {
  return withAndroidManifest(config, (config) => {
    config.modResults.manifest.queries = [
      {
        intent: [
          {
            action: [{ $: { "android:name": "android.intent.action.SENDTO" } }],
            data: [{ $: { "android:scheme": "mailto" } }],
          },
          {
            action: [{ $: { "android:name": "android.intent.action.DIAL" } }],
          },
        ],
      },
    ];

    return config;
  });
};

module.exports = withAndroidQueries;
```

> You can then [import the custom config plugin](/config-plugins/plugins-and-mods/#import-a-plugin) in your project's app config. This example plugin will enable linking to phone and email apps.

Reviewer: ```suggestion
You can then [import the custom config plugin](/config-plugins/plugins-and-mods/#import-a-plugin) in your project's app config.
```

| `tel`            | Open phone app, eg: `tel:+123456789`          |
| `sms`            | Open SMS app, eg: `sms:+123456789`            |

On newer Android versions, include the appropriate queries in the **AndroidManifest.xml** to open links. This can be done by [creating a config plugin](/config-plugins/plugins-and-mods/#create-a-plugin). For example:

```js my-plugin.js
const { withAndroidManifest } = require("@expo/config-plugins");

const withAndroidQueries = (config) => {
  return withAndroidManifest(config, (config) => {
    config.modResults.manifest.queries = [
      {
        intent: [
          {
            action: [{ $: { "android:name": "android.intent.action.SENDTO" } }],
            data: [{ $: { "android:scheme": "mailto" } }],
          },
          {
            action: [{ $: { "android:name": "android.intent.action.DIAL" } }],
          },
        ],
      },
    ];

    return config;
  });
};

module.exports = withAndroidQueries;
```

You can then [import the custom config plugin](/config-plugins/plugins-and-mods/#import-a-plugin) in your project's app config. This example plugin will enable linking to phone and email apps.

> Enables linking to the phone and email app.

Reviewer: ```suggestion
```
